src/AzureExtension/Helpers/Json.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Newtonsoft.Json;
+
+namespace DevHomeAzureExtension.Helpers;
+
+public static class Json
+{
+    public static async Task<T> ToObjectAsync<T>(string value)
+    {
+        if (typeof(T) == typeof(bool))
+        {
+            return (T)(object)bool.Parse(value);
+        }
+
+        return await Task.Run<T>(() =>
+        {
+            return JsonConvert.DeserializeObject<T>(value)!;
+        });
+    }
+
+    public static async Task<string> StringifyAsync<T>(T value)
+    {
+        if (typeof(T) == typeof(bool))
+        {
+            return value!.ToString()!.ToLowerInvariant();
+        }
+
+        return await Task.Run<string>(() =>
+        {
+            return JsonConvert.SerializeObject(value);
+        });
+    }
+
+    public static string Stringify<T>(T value)
+    {
+        if (typeof(T) == typeof(bool))
+        {
+            return value!.ToString()!.ToLowerInvariant();
+        }
+
+#pragma warning disable CA1869 // Cache and reuse 'JsonSerializerOptions' instances
```

Comment:
Why not just reuse the JsonSerializerOptions object?

Reply:
Same situation as LocalSettings, this was copied from github etxension for simpler settings UI implementation.  I added the warning suppress to clean up the build but otherwise didn't touch it. If we want to clean up this code it should be cleaned in both repositories, and I think that should be a separate set of very similar PRs. 

see: https://github.com/microsoft/devhomegithubextension/blob/main/src/GitHubExtension/Helpers/Json.cs   It's the same code, just without the warning suppress.